### ISSUE TITLE
Change link relationship label.

### ISSUE THREAD

**Issue by Soean** [WordPress/gutenberg#20208]
The label `Link rel` is used in the image URL input and in the button block to add link [relationship types](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types) to the links.

<img width="285" alt="Bildschirmfoto 2020-02-13 um 14 19 16" src="https://user-images.githubusercontent.com/695201/74410520-db8adf00-4e6b-11ea-895f-70f24fd142ee.png">

<img width="277" alt="Bildschirmfoto 2020-02-13 um 14 17 54" src="https://user-images.githubusercontent.com/695201/74410463-b4cca880-4e6b-11ea-943c-653f7032293a.png">

I would suggest to change the label to `Link relationship`, `Relation attribute` or `Relationship attribute`. Or should we add a help text? Should we add a link to a help page?

**Comment by karmatosed** [comment]
I agree this is super confusing, let's get it fixed! Thanks for reporting.

**Comment by karmatosed** [comment]
> I would suggest to change the label to Link relationship, Relation attribute or Relationship attribute. Or should we add a help text? Should we add a link to a help page?

I like idea of some explanation and saying the full term.

**Comment by skorasaurus** [comment]
This was also discussed at #29141 

There were some great discussions and suggestions there by @SukcesStrony @jameskoster and @jarekmorawski.  

To recap them from there: 

1.     Move the "Link rel" input to the "Advanced" section [as mentioned earlier](https://github.com/WordPress/gutenberg/issues/29141#issuecomment-788871585)
2.     Add helper text to clarify what this input is actually for
 
Perhaps maybe also hyperlinking "link relation" to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel in the description text?

**Comment by coderGtm** [comment]
Hi all
I have created a [PR](https://github.com/WordPress/gutenberg/pull/70451) to bring this to life! Please have a look and propose your feedback. Also, the help text that I added can be made more clear imo but wasn't aware of anything better atm.

**Comment by karmatosed** [comment]
Removing label as looks like we certainly have a direction.

**Comment by Mamaduka** [comment]
Reposting from the PR.

I think this falls under the "Need Decision" label. I don't have a strong opinion; both labels work for me. Considering it's an advanced setting, users shouldn't just fill it out.

Additionally, the terminology needs to be consistent across the project. The image link control isn't the only place with this field. Examples:

* https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop+%22%27Link+rel%27%22&type=code
* https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%22%27Link+rel%27%22&type=code

I would hold off making further changes until there's a final decision.
